src/components/SingleName/DetailsItemEditable.js

```diff
@@ -380,46 +380,45 @@ const Editable = ({
   )
 }
 
-class DetailsEditable extends Component {
-  _renderViewOnly() {
-    let { value, keyName, type, deedOwner, isDeedOwner, domain } = this.props
-    if (parseInt(value, 16) === 0) {
-      let [newValue, newType] = getDefaultMessage(keyName)
-      value = newValue
-      type = newType
-      if (
-        keyName === 'Owner' &&
-        domain.parent === 'eth' &&
-        parseInt(deedOwner, 16) !== 0
-      ) {
-        value = 'Pending'
-        if (isDeedOwner) {
-          value += '(You have not finalised)'
-        }
+function ViewOnly({ value, keyName, type, deedOwner, isDeedOwner, domain }) {
+  if (parseInt(value, 16) === 0) {
+    let [newValue, newType] = getDefaultMessage(keyName)
+    value = newValue
+    type = newType
+    if (
+      keyName === 'Owner' &&
+      domain.parent === 'eth' &&
+      parseInt(deedOwner, 16) !== 0
+    ) {
+      value = 'Pending'
+      if (isDeedOwner) {
+        value += '(You have not finalised)'
```

Comment:
Is this working as expected? I expected that I see "You have not finalised" under `/name/auctionednofinalise.eth`
<img width="1235" alt="Screenshot 2019-04-29 at 10 25 29" src="https://user-images.githubusercontent.com/2630/56887302-29636180-6a69-11e9-82f6-ff170adfc1c6.png">